src/Stripe/Infrastructure/ParameterBuilder.cs

```diff
@@ -110,7 +107,7 @@ public static string ApplyParameterToUrl(string url, string argument, string val
             if (!url.Contains("?"))
                 token = "?";
 
-            return string.Format("{0}{1}{2}={3}", url, token, argument, HttpUtility.UrlEncode(value));
+            return string.Format("{0}{1}{2}={3}", url, token, argument, WebUtility.UrlEncode(value));
```

Comment:
no issues with the current project with this? did you run the test suite by chance?